### DISCUSSION TITLE
Welcome to Hospital_Management_System Discussions!

### DISCUSSION THREAD

**Discussion by Jahnavi-Anand** [Jahnavi-Anand/Hospital_Management_System#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jahnavi-Anand** [comment]
Hii

**Reply by oringejooz** [reply]
Hello 😄

**Comment by Aanidhay** [comment]
![WhatsApp Image 2023-10-31 at 22 33 26_40652077](https://github.com/VesperSK/Hospital_Management_System/assets/115068672/fd9bc301-e7e7-43f2-8bba-4cb3dac0a677)

Well done guys!!!

**Comment by oringejooz** [comment]
Front End Team -Home Page & Admin Login Done

![image](https://github.com/Jahnavi-Anand/Hospital_Management_System/assets/124855325/0f72972a-1d8b-4d5f-b39e-99a6eb836732)

👍🏻

**Comment by oringejooz** [comment]
Please use Discussions. 
It is mandatory

**Comment by oringejooz** [comment]
Also if you are having trouble with your Wamp server then tell me okay.

**Comment by oringejooz** [comment]
the admin/ajax_job_request.php file has a lot of issues Kavya, kindly see to it

**Reply by oringejooz** [reply]
Issue resolved just make the connection 😸

**Comment by Aanidhay** [comment]
![WhatsApp Image 2023-11-13 at 15 40 56_50e93ec6](https://github.com/Jahnavi-Anand/Hospital_Management_System/assets/115068672/3ce201ea-0e42-4504-9792-f95fbbeddfc7)
Discussed further about the project on 5th Nov, 2023

**Comment by Aanidhay** [comment]
![WhatsApp Image 2023-11-11 at 23 29 58_e424ccfb](https://github.com/Jahnavi-Anand/Hospital_Management_System/assets/115068672/f366a564-efbc-4620-a7aa-275fd37b5b86)
Discussed the remaining project with all the members on 11th Nov, 2023

**Comment by Aanidhay** [comment]
![WhatsApp Image 2023-11-16 at 22 27 51_607008f5](https://github.com/Jahnavi-Anand/Hospital_Management_System/assets/115068672/9a9a8e77-0ef8-49e2-9419-69d935aa7fee)
Redistributed the work among all the members of the group on 16th Nov, 2023
